src/main/java/org/alliancegenome/curation_api/dao/GenomicEntityDAO.java

```diff
@@ -0,0 +1,15 @@
+package org.alliancegenome.curation_api.dao;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.alliancegenome.curation_api.dao.base.BaseSQLDAO;
+import org.alliancegenome.curation_api.model.entities.GenomicEntity;
+
+@ApplicationScoped
+public class GenomicEntityDAO extends BaseSQLDAO<GenomicEntity> {
+
```

Comment:
What do you need this class for?

Reply:
It's used in the validation code for ConstructGenomicEntity associations.  It's the first time we've had a GenomicEntity field and not the BiologicalEntity parent class or one of the child classes.